azure-cognitiveservices-vision-computervision/setup.py

```diff
@@ -52,7 +52,8 @@
     name=PACKAGE_NAME,
     version=version,
     description='Microsoft Azure {} Client Library for Python'.format(PACKAGE_PPRINT_NAME),
-    long_description=readme + '\n\n' + history,
+    long_description=readme,
```

Comment:
Beware that you concat a MD and a RST file here, you want to convert the HISTORY to MD as well

Reply:
@diberry You might be able to use [Pandoc](https://pandoc.org/try/) for this.

Cc: @lmazuel

Reply:
The History is so simple that a quick search/replace should be enough :). History is really level 1 title and bullet list, that's it. But Pandoc would work indeed (I feel it's nuclear power where a hammer is enough, that's all ;))